src/compas/datastructures/mesh/operations/split.py

```diff
@@ -193,7 +193,7 @@ def mesh_split_face(mesh, fkey, u, v):
     >>> mesh = Mesh.from_obj(compas.get("faces.obj"))
     >>> fkey = mesh.get_any_face()
     >>> # u and v defines the new edge after splitting
-    >>> u = mesh.get_any_face_vertex(fkey)
+    >>> u = mesh.face_vertices(fkey)[1]
```

Comment:
why not `[0]`?

Reply:
true, no idea why i used 1, changing to `0`